app/controllers/projects_controller.rb

```diff
@@ -164,10 +164,6 @@ def update
   # DELETE /projects/1.json
   def destroy
     @project.destroy
```

Comment:
I think you need to put in some safe guard here. only destroy the project wheren there are no samples under it.

Reply:
ok